profiles/base.yaml

```diff
@@ -402,6 +402,8 @@ hunting_room_strict_mana: false
 hunting_room_max_searches: 30
 hunting_file_list:
 stop_on_low_threshold: 2
+# sets a default stop_on threshold for hunting-buddy.lic 
+stop_on_threshold: 32
```

Comment:
What do you think about naming this `stop_on_high_threshold` to make it clearer how this differs from the setting above it?

Reply:
sounds good, will make the changes.